src/ss/scripts/AEpickableTemplate.mel

```diff
@@ -4,7 +4,7 @@ global proc AEpickableTemplate(string $nodeName)
   editorTemplate -beginLayout "Geometry" -collapse 0;
   editorTemplate -addControl "rotate";
   editorTemplate -addControl "offset";
-  editorTemplate -addControl -label "Shape" "shape" "swapShape";
+  editorTemplate -addControl -label "Shape" "shape";
```

Comment:
This line could be simplified to...

```python
editorTemplate -addControl "shape";
```

The control will be created with _"Shape"_ capitalised label by default.